config/config.go

```
@@ -85,18 +90,23 @@ func Load(configFilePath string) (Config, error) {
 	}
 
 	file, err := os.OpenFile(configFilePath, os.O_RDONLY, 0644)
+
 	if err != nil {
 		if os.IsNotExist(err) {
 			return config, fmt.Errorf("could not find config file at path %s", configFilePath)
 		}
 		return config, err
 	}
-	defer file.Close()
 
 	if err := json.NewDecoder(file).Decode(&config); err != nil && !errors.Is(err, io.EOF) {
+
 		return config, err
 	}
 
+	// reset pointer for computing hash
+	file.Seek(0, io.SeekStart)
+	config.FileHash, _ = getFileHash(file)
```

curious as to why the error is ignored?

---

added error logging, thanks